docs/frameworks/sveltekit.md

```diff
@@ -0,0 +1,229 @@
+---
+title: SvelteKit | Frameworks
+---
+
+# SvelteKit
+
+> For `Type declarations`, `Prompt for update` and `Periodic SW Updates` go to [Svelte](/frameworks/svelte.html) entry.
+> 
+
+You should remove all references to [SvelteKit service worker module](https://kit.svelte.dev/docs#modules-$service-worker) <outbound-link />
+to disable it on your application.
+
+Since `SvelteKit` will use `SSR / SSG`, we need to add the `ReloadPrompt` component using `dynamic import`. `Vite Plugin PWA`
+will only register the service worker on build, it is aligned with the current behavior of
+[SvelteKit service worker module](https://kit.svelte.dev/docs#modules-$service-worker) <outbound-link />.
+
+The best place to include the `ReloadPrompt` is on the main layout of the application:
+
+<details>
+  <summary><strong>src/routes/__layout.svelte</strong> code</summary>
+
+```html
+<script>
+  import { onMount } from 'svelte'
+  import { browser, dev } from '$app/env'
+
+  let ReloadPrompt
+  onMount(async () => {
+    !dev && browser && (ReloadPrompt = (await import('$lib/components/ReloadPrompt.svelte')).default)
+  })
+</script>
+
+<svelte:head>
+  {#if (!dev && browser)}
+  <link rel="manifest" href="/_app/manifest.webmanifest">
+  {/if}
+</svelte:head>
+
+<main>
+  <slot />
+</main>
+
+{#if ReloadPrompt}
+<svelte:component this={ReloadPrompt} />
+{/if}
+```
+</details>
+
+## SvelteKit Adapters
+
+The main problem with the current implementation of the service worker module of `SvelteKit` is that you don't have access
```

Comment:
I would recommend against manually wrapping the text. If you add a word anywhere it means you need to update every subsequent line. I think it's much easier to manage as one long line. Text editors can always handle wrapping the text automatically